htdocs/install/mysql/migration/12.0.0-13.0.0.sql

```diff
@@ -330,3 +330,6 @@ ALTER TABLE llx_bank ADD COLUMN origin_type varchar(64) NULL;
 ALTER TABLE llx_bank ADD COLUMN import_key varchar(14);
 
 ALTER TABLE llx_menu MODIFY COLUMN enabled text;
+
+ALTER TABLE llx_website_page ADD COLUMN fk_typeobject varchar(255);
```

Comment:
To match the other case (resources, element, when a link to an object with a type is defined, can you rename field "fk_typeobject" into "object_type" (we must use fk_ only when value is a foreign id into another table) and here the type is a string to define type of object.
Like done into resource, element and some other table (even if in past we did a lot of different way, we should try now to nomalize how we name field).

Reply:
Ok, I'll change the name of the field.
For example:
If I want to create a page using a Dolibarr object, it allows loading the object class and the desired id. I followed the syntax of an extrafield as a list object from a table.